config/gasSchedule.go

```diff
@@ -103,12 +105,22 @@ func CreateGasConfig(gasMap GasScheduleMap) (*GasCost, error) {
 		return nil, err
 	}
 
-	dynamicStorageLoadParams := &DynamicStorageLoad{}
-	err = mapstructure.Decode(gasMap["DynamicStorageLoad"], dynamicStorageLoadParams)
+	dynamicStorageLoadUnsigned := &DynamicStorageLoadUnsigned{}
+	err = mapstructure.Decode(gasMap["DynamicStorageLoad"], dynamicStorageLoadUnsigned)
 	if err != nil {
 		return nil, err
 	}
 
+	a := getSignedCoefficient(dynamicStorageLoadUnsigned.A, dynamicStorageLoadUnsigned.SignOfA)
```

Comment:
can you extract this conversion from unsigned data struct to signed data struct into a function and call it here?

Reply:
Done.